core/shared/src/main/scala/zio/internal/FiberRuntime.scala

```diff
@@ -1182,6 +1185,7 @@ final class FiberRuntime[E, A](fiberId: FiberId.Runtime, fiberRefs0: FiberRefs,
     }
 
     reportExitValue(e)
+    getSupervisor().onObserverNotify(e, self)
```

Comment:
We are guarding against and ignoring exceptions thrown by loggers https://github.com/zio/zio/blob/series/2.x/core/shared/src/main/scala/zio/internal/FiberRuntime.scala#L1164. Should the same be done for supervisors?